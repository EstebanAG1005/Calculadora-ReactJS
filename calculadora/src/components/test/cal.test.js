import renderer from 'react-test-renderer';
import Calculadora from '../Calculadora'
import React from 'react';


it('Si el display de la calculadora realiza un render correcto', () => {
    const tree = renderer
      .create(<Calculadora value="28"></Calculadora>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
