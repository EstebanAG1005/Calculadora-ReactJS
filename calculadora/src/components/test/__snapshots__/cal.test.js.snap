// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CalcDisplay component renders correctly 1`] = `
<div
  className="calculator"
>
  <div
    className="calculator__result"
  >
    <div
      className="calculator__result__exp"
    />
    <div
      className="calculator__result__exp"
    />
  </div>
  <div
    className="calculator__btns"
  >
    <button
      className="btn__op"
      onClick={[Function]}
      type="button"
    >
      C
    </button>
    <button
      className="btn__op"
      onClick={[Function]}
      type="button"
    >
      (
    </button>
    <button
      className="btn__op"
      onClick={[Function]}
      type="button"
    >
      )
    </button>
    <button
      className="btn__op"
      onClick={[Function]}
      type="button"
    >
      /
    </button>
    <button
      className=""
      onClick={[Function]}
      type="button"
    >
      7
    </button>
    <button
      className=""
      onClick={[Function]}
      type="button"
    >
      8
    </button>
    <button
      className=""
      onClick={[Function]}
      type="button"
    >
      9
    </button>
    <button
      className="btn__op"
      onClick={[Function]}
      type="button"
    >
      x
    </button>
    <button
      className=""
      onClick={[Function]}
      type="button"
    >
      4
    </button>
    <button
      className=""
      onClick={[Function]}
      type="button"
    >
      5
    </button>
    <button
      className=""
      onClick={[Function]}
      type="button"
    >
      6
    </button>
    <button
      className="btn__op"
      onClick={[Function]}
      type="button"
    >
      -
    </button>
    <button
      className=""
      onClick={[Function]}
      type="button"
    >
      1
    </button>
    <button
      className=""
      onClick={[Function]}
      type="button"
    >
      2
    </button>
    <button
      className=""
      onClick={[Function]}
      type="button"
    >
      3
    </button>
    <button
      className="btn__op"
      onClick={[Function]}
      type="button"
    >
      +
    </button>
    <button
      className=""
      onClick={[Function]}
      type="button"
    >
      dark mode
    </button>
    <button
      className=""
      onClick={[Function]}
      type="button"
    >
      0
    </button>
    <button
      className=""
      onClick={[Function]}
      type="button"
    >
      .
    </button>
    <button
      className="btn__op"
      onClick={[Function]}
      type="button"
    >
      =
    </button>
  </div>
</div>
`;

exports[`Si el display de la calculadora realiza un render correcto 1`] = `
<div
  className="calculator"
>
  <div
    className="calculator__result"
  >
    <div
      className="calculator__result__exp"
    />
    <div
      className="calculator__result__exp"
    />
  </div>
  <div
    className="calculator__btns"
  >
    <button
      className="btn__op"
      onClick={[Function]}
      type="button"
    >
      C
    </button>
    <button
      className="btn__op"
      onClick={[Function]}
      type="button"
    >
      (
    </button>
    <button
      className="btn__op"
      onClick={[Function]}
      type="button"
    >
      )
    </button>
    <button
      className="btn__op"
      onClick={[Function]}
      type="button"
    >
      /
    </button>
    <button
      className=""
      onClick={[Function]}
      type="button"
    >
      7
    </button>
    <button
      className=""
      onClick={[Function]}
      type="button"
    >
      8
    </button>
    <button
      className=""
      onClick={[Function]}
      type="button"
    >
      9
    </button>
    <button
      className="btn__op"
      onClick={[Function]}
      type="button"
    >
      x
    </button>
    <button
      className=""
      onClick={[Function]}
      type="button"
    >
      4
    </button>
    <button
      className=""
      onClick={[Function]}
      type="button"
    >
      5
    </button>
    <button
      className=""
      onClick={[Function]}
      type="button"
    >
      6
    </button>
    <button
      className="btn__op"
      onClick={[Function]}
      type="button"
    >
      -
    </button>
    <button
      className=""
      onClick={[Function]}
      type="button"
    >
      1
    </button>
    <button
      className=""
      onClick={[Function]}
      type="button"
    >
      2
    </button>
    <button
      className=""
      onClick={[Function]}
      type="button"
    >
      3
    </button>
    <button
      className="btn__op"
      onClick={[Function]}
      type="button"
    >
      +
    </button>
    <button
      className=""
      onClick={[Function]}
      type="button"
    >
      dark mode
    </button>
    <button
      className=""
      onClick={[Function]}
      type="button"
    >
      0
    </button>
    <button
      className=""
      onClick={[Function]}
      type="button"
    >
      .
    </button>
    <button
      className="btn__op"
      onClick={[Function]}
      type="button"
    >
      =
    </button>
  </div>
</div>
`;
